### DISCUSSION TITLE
is it possible to close other os_windows but keep the active os window?

### DISCUSSION THREAD

**Discussion by tg0h** [kovidgoyal/kitty#7113] [General]
Similar to `close_other_windows_in_tab` or `close_other_tabs_in_os_window`, when i have several kitty os windows open, is it possible to close all the other os windows which are not active?

**Comment by tg0h** [comment]
my workaround is to use yabai and skhd to close all non-active kitty os windows

```bash
# skhd config
lshift + lalt + lcmd - w : yabai -m query --windows | jq '.[] | select (."has-focus"==false and .app=="kitty") | .id' | xargs -I {} -P 0 yabai -m window --close {}
```

**Comment by chmouel** [comment]
Fixed in 0.32.2 fyi

**Reply by tg0h** [reply]
nice! thanks for the headsup

**Comment by tg0h** [comment]
this is now provided via https://sw.kovidgoyal.net/kitty/actions/#action-close_other_os_windows
